pkg/driver/aws-ebs/aws_ebs.go

```diff
@@ -433,3 +477,21 @@ func withKMSKeyHook(c *clients.Clients) csistorageclasscontroller.StorageClassHo
 	}
 	return hook
 }
+
+func withVolumeAttributesClassHook(c *clients.Clients) (dc.DeploymentHookFunc, []factory.Informer) {
+	hook := func(_ *opv1.OperatorSpec, deployment *appsv1.Deployment) error {
+		newArg := "--feature-gates=VolumeAttributesClass=true"
+		for i := range deployment.Spec.Template.Spec.Containers {
+			container := &deployment.Spec.Template.Spec.Containers[i]
+			if container.Name == "csi-provisioner" || container.Name == "csi-resizer" {
+				container.Args = append(container.Args, newArg)
+			}
+		}
+
+		return nil
+	}
+	informers := []factory.Informer{
+		c.GetInfraInformer().Informer(),
+	}
```

Comment:
I think this hook does not need any informer. It does not react to any API object change.